### ISSUE TITLE
Examine: Disassembly and raw

### ISSUE THREAD

**Issue by maxashen** [jakimfett/MMPLv2#34]
Can the user examine raw bytecode (I believe this is implied as it is technically required)? Can the user disassemble the bytecode (eg: javap)?

**Comment by jakimfett** [comment]
Yes, both of those are covered under the "[Right to Examine](https://github.com/jakimfett/MMPLv2/blob/master/LICENSE.md#4-right-to-examine)", but I'll think about ways to make it more explicitly permissable.
